.gitignore

```diff
@@ -1,6 +1,7 @@
 *.pyc
 *.swpi
 *.cache
+*.coveragerc
```

Comment:
We *do* want the coveragerc file, we don't want the coverage file.  If you look you'll find that the coveragerc file has configuration information about using coverage (a tool to test which lines are covered by tests) and that the coverage file has a bunch of data for the last particular run of the coverage tool.